test/spawn_ecsact_cli.hh

```diff
@@ -0,0 +1,6 @@
+#pragma once
+
+#include <string>
+#include <vector>
+
+int spawn_ecsact_cli(std::string executable, std::vector<std::string> args);
```

Comment:
nit: could we add a namespace to this fn? maybe `ecsact::entt::test::detail` or something. I don't care too much what it is, but we should avoid adding functions/classes without namespaces.